src/njoy21/input/HEATR/Card5a/Qbar.hpp

```diff
@@ -0,0 +1,17 @@
+struct Qbar {
+  using Value_t = ENDFtk::TAB1;
+  static std::string name(){ return "qbar"; }
+  static std::string description(){
+    return
+    "The qbar input is a endf format tab1 record giving qbar vs. energy.\n"
```

Comment:
avoid math in english for documentation. rather than saying `qbar vs energy`, how about something like `energy released per reaction as a function of incident particle energy`